extensions/basic/README.md

```diff
@@ -0,0 +1,8 @@
+## JRuby Extensions basic concepts
+
+This project aims to demo small and easy concepts around jruby extensions, this concepts will be listed as
+
+* Classes and Modules
+* Static methods in both elements
+* Loading the extensions
+* ...
```

Comment:
Yeah! I was more on the idea shared later here by @enebo, using BasicLibraryService, using Library, etc... but yeah! the naming is funny!